service/src/test/java/bio/terra/cbas/controllers/VerifyPactsAllControllers.java

```diff
@@ -45,13 +46,18 @@
 import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
 import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.TestPropertySource;
 import org.springframework.test.web.servlet.MockMvc;
 
 @WebMvcTest
-@ContextConfiguration(classes = {RunSetsApiController.class, CbasApiConfiguration.class})
+@ContextConfiguration(
+    classes = {RunSetsApiController.class, MethodsApiController.class, CbasApiConfiguration.class})
+@TestPropertySource(properties = "cbas.cbas-api.runSetsMaximumRecordIds=100")
+@TestPropertySource(properties = "cbas.cbas-api.maxWorkflowInputs=100")
+@TestPropertySource(properties = "cbas.cbas-api.maxWorkflowOutputs=40")
```

Comment:
these configurations were needed before, but their absence was masked by the fact that none of the pacts that need them were ever being verified.